include/versions.h

```diff
@@ -16,4 +16,27 @@
 #define GC_EU_MQ      12
 #define GC_JP_CE      13
 
+// NTSC/PAL
+#if OOT_VERSION == PAL_1_0 || OOT_VERSION == PAL_1_1 || OOT_VERSION == GC_EU || OOT_VERSION == GC_EU_MQ || OOT_VERSION == GC_EU_MQ_DBG
+#define OOT_NTSC 0
+#define OOT_PAL 1
+#else
+#define OOT_NTSC 1
+#define OOT_PAL 0
+#endif
+
+// PAL N64 (50 Hz)
+#if OOT_VERSION == PAL_1_0 || OOT_VERSION == PAL_1_1
+#define OOT_PAL_N64 1
+#else
+#define OOT_PAL_N64 0
+#endif
+
+// Master Quest
+#if OOT_VERSION == GC_JP_MQ || OOT_VERSION == GC_US_MQ || OOT_VERSION == GC_EU_MQ || OOT_VERSION == GC_EU_MQ_DBG
+#define OOT_MQ 1
+#else
+#define OOT_MQ 0
+#endif
```

Comment:
I wouldnt have a probelm with phasing the OOT_ outslowly and doing it where it makes sense, for example MASTER_QUEST. but it may make sense to also not complicate the diff of this PR. whatever wrorks